runners/pvc_autoresizer_test.go

```diff
@@ -81,4 +89,143 @@ var _ = Describe("test resizer", func() {
 			}
 		})
 	})
+
+	Context("resize", func() {
+		Context("parameter tests", func() {
+			ctx := context.Background()
+			pvcNS := "default"
+			increase := "1Gi"
+			limit := int64(100 << 30)
+			volumeMode := corev1.PersistentVolumeFilesystem
+
+			testCases := []struct {
+				description   string
+				pvcSizeGi     int64
+				expectSizeGi  int64
+				threshold     string
+				availableByte int64
+			}{
+				{
+					description:   "Should resize(absolute value)",
+					pvcSizeGi:     10,
+					expectSizeGi:  11,
+					threshold:     "5Gi",
+					availableByte: 5<<30 - 1,
+				},
+				{
+					description:   "Should not resize(absolute value)",
+					pvcSizeGi:     10,
+					expectSizeGi:  10,
+					threshold:     "5Gi",
+					availableByte: 5 << 30,
+				},
+				{
+					description:   "Should resize(%)",
+					pvcSizeGi:     10,
+					expectSizeGi:  11,
+					threshold:     "50%",
+					availableByte: 5<<30 - 1,
+				},
+				{
+					description:   "Should not resize(%)",
+					pvcSizeGi:     10,
+					expectSizeGi:  10,
+					threshold:     "50%",
+					availableByte: 5 << 30,
+				},
+			}
+
+			counter := 0
+			for _, tc := range testCases {
+				counter++
```

Comment:
This can be `for counter, tc := range testCases {`.

Reply:
I improved it.
https://github.com/topolvm/pvc-autoresizer/pull/48/commits/0d7c4c2b5648fa364809c11ec781308eea0d94e8

Reply:
I fixed it.
https://github.com/topolvm/pvc-autoresizer/pull/48/commits/767cad4613d84a1efda9baea0447f98c762998ab